net/dns/manager.go

```diff
@@ -331,6 +331,8 @@ func (m *Manager) NextPacket() ([]byte, error) {
 	return buf, nil
 }
 
+const dnsQueryTimeout = 10 * time.Second
+
 func (m *Manager) Query(ctx context.Context, bs []byte, from netaddr.IPPort) ([]byte, error) {
```

Comment:
// Query executes a DNS query (encoded in bs) from the provided IPPort.
// It's called for requests arriving over TCP and UDP.
// The bs query is in _____ form (is it always UDP form, TCP form, varies depending on transport type?).

Reply:
Done